sorting/mergesort.js

```diff
@@ -0,0 +1,49 @@
+/*
+Merge Sort is  based on the principle of Divide and Conquer Algorithm.
```

Comment:
Merge sort in javascript already exists, kindly remove it and do 
"git pull origin main"